src/Features/CSharp/Portable/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -202,6 +304,18 @@ static TypeSyntax TryGetSingleTypeArgument(TypeSyntax type)
                 }
                 return null;
             }
+
+            static TypeSyntax ParameterTypeSyntax(IParameterSymbol parameterSymbol)
+            {
+                if (parameterSymbol is object &&
+                    parameterSymbol.DeclaringSyntaxReferences[0].GetSyntax() is ParameterSyntax parameterSyntax &&
+                    parameterSymbol.ContainingSymbol is IMethodSymbol method &&
+                    method.GetAllMethodSymbolsOfPartialParts().Length == 1)
```

Comment:
what if you are in the partial case?

Reply:
`// We don't handle partial methods yet`

More generally, we don't cascade this annotation change yet. If you change the nullability of an override/implementation, there are cases where you'll need to propagate to a base class or interface. Currently, you must do that by hand.